app/src/main/java/nu/yona/app/api/manager/network/BaseImpl.java

```diff
@@ -155,6 +156,30 @@ public void onFailure(retrofit2.Call call, Throwable t) {
         };
     }
 
+    /**
+     * Gets call.
+     *
+     * @param listener the listener
+     * @return the call
+     */
+    Callback getCall(final DataLoadListenerImpl listener) {
```

Comment:
Please rename in ``createCallback``. Get suggest it already exists and you're making a callback, not a call.